django-stubs/contrib/postgres/operations.pyi

```diff
@@ -1,9 +1,29 @@
+from typing import Dict, Literal, Tuple
+
+from django.db.backends.base.schema import BaseDatabaseSchemaEditor
+from django.db.migrations import AddConstraint, AddIndex, RemoveIndex
 from django.db.migrations.operations.base import Operation
+from django.db.migrations.state import ProjectState
+from django.db.models.constraints import BaseConstraint
 
 class CreateExtension(Operation):
     reversible: bool = ...
     name: str = ...
     def __init__(self, name: str) -> None: ...
+    def state_forwards(self, app_label: str, state: ProjectState) -> None: ...
+    def database_forwards(
+        self, app_label: str, schema_editor: BaseDatabaseSchemaEditor, from_state: ProjectState, to_state: ProjectState
+    ) -> None: ...
+    def database_backwards(
+        self, app_label: str, schema_editor: BaseDatabaseSchemaEditor, from_state: ProjectState, to_state: ProjectState
+    ) -> None: ...
+    def extension_exists(self, schema_editor: BaseDatabaseSchemaEditor, extension: str) -> bool: ...
+    def describe(self) -> str: ...
+    @property
+    def migration_name_fragment(self) -> str: ...
```

Comment:
Looks like this is defined in `Operation`: https://github.com/django/django/blob/67d0c4644acfd7707be4a31e8976f865509b09ac/django/db/migrations/operations/base.py#L82-L88

Can you please make sure that all parent methods stubs are not redefined?

Reply:
OK. Updated.